src/observers/OmnisharpStatusBarObserver.ts

```diff
@@ -12,6 +12,10 @@ export class OmnisharpStatusBarObserver extends BaseStatusBarItemObserver {
             case OmnisharpServerOnServerError.name:
                 this.SetAndShowStatusBar('$(flame)', 'o.showOutput', 'rgb(218,0,0)', 'Error starting OmniSharp');
                 break;
+            case OmnisharpServerOnStdErr.name:
+                let msg = (<OmnisharpServerOnStdErr>event).message;
+                this.SetAndShowStatusBar('$(flame)', 'o.showOutput', 'rgb(218,0,0)', `OmniSharp process errored:${msg}`);
```

Comment:
Can you extract out constants for the colors and call them names like "red" and "green"?